README.md

```diff
@@ -338,6 +345,25 @@ The options `-h`, `-i`, `-r`, `-f` are exclusive. The options `-s` and `-k` are
 at the LPC (where each interactive node has its own scheduler). The script can ssh to each node and run itself to modify the jobs
 on that node (because each scheduler can only be accessed for write operations from its respective node).
 
+The option `-X` will resubmit the jobs (just like `-s`), except that it will tell the job to get its input file from a specific site based on the list of sites where that file
+is located and some user preferences. Therefore, the options `-X` and `-s` are exclusive, as are the options `-X` and `-k`. The options `-C`, `-D`, `-K`, `-L`, `-U`, and `-V`
+are only applicable when using the option `-X`. There are two ways in which the program can find the appropriate input file(s) for the job:
+1. specify an HTCondor ClassAd name, where the ClassAd contains a comma separated list of files (`-C`).
+2. specify the location of a set of log files and a key with which to parse the logs files (`-L`/`-K`). It is expected that the log will contain a single line with a comma separated list of files.
+This is similar in spirit to using the `-x` option, except in that case all of the resubmitted jobs will try to access the input files from a user specified location. In this case,
+the user doesn't specify a particular location, but a set of loose preferences (more on that in a moment). If both the `-X` and `-x` options are specified, then the jobs will preferentially
+read their input from the site found using the automated mechanism. However, if no suitable site can be found, then the job will be resubmitted using the xrootd redirector (or site name) specified
+by the `-x` option.
+
+When using the `-X` option, a user may indicate a preference for specific sites in two ways:
+1. The user can broadly prefer sites located in the United States using the `-U` option
+2. The user can add a list of sites to the `.prodconfig` file of the form:
```

Comment:
for consistency with the rest of the docs, just note here that `preferredsites` can be provided via `.prodconfig` and then add it to the `Configuration` section below

Reply:
Isn't the configuration section below only for mandatory configuration parameters? I put the information in the job management section because it's an optional configuration.

Reply:
No, no distinction is made there between mandatory or optional parameters.

Reply:
Changed